types/node/ts3.2/globals.d.ts

```diff
@@ -6,3 +6,10 @@ declare namespace NodeJS {
         bigint(): bigint;
     }
 }
+
+interface Buffer extends Uint8Array {
+    readBigUInt64BE(offset?: number): bigint;
```

Comment:
`readBigInt64BE`, `readBigInt64LE`, `writeBigUInt64BE` and `writeBigUInt64LE` seem to be missing here.